src/cljs/athens/views/blocks/textarea_keydown.cljs

```diff
@@ -111,40 +111,43 @@
 
 (defn filter-slash-options
   [query]
-  (if (blank? query)
-    slash-options
-    (filterv (fn [[text]]
-               (re-find (re-pattern (str "(?i)" query)) text))
-             slash-options)))
-
+  ((try
+     (if (blank? query)
+       slash-options
+       #(filterv (fn [[text]]
+                   (re-find (re-pattern (str "(?i)" query)) text))
+                 slash-options))
+     (catch js/Error _
+       slash-options))))
```

Comment:
this just silences the problem and does not really fix it :/

also this will silence *any* JavaScript error, which is not a good idea.

this is basically a query for "does `text` contain `query`, case-insensitive", right? could we maybe just do `(str/includes? (st/lower-case text) (st/lower-case query))`?

Reply:
(see: https://clojuredocs.org/clojure.string/includes_q, https://clojure.github.io/clojure/clojure.string-api.html)

Reply:
to expand: silencing broad classes of errors is a bad idea, because it can mask other problems and make them harder to notice and debug

Reply:
Thank you for your review :-)

I completely agree that silencing errors is not the greatest approach. My main concern here was that typing an invalid Regex shouldn't result in a js/alert being shown to the end user.

I also agree the `str/includes?` approach would probably solve this problem in a fancier way and don't really ser any downsides in using `str/includes?` instead of `re-find` here. If we decide on going this way I can make a new commit and fix it!